src/styles/utils/_mixins.scss

```diff
@@ -4,3 +4,50 @@
     #{$_property}: $_toValue;
   }
 }
+
+@mixin Desktop {
```

Comment:
It's a good practice to name mixins with lowercase and hyphens, like 'desktop' instead of 'Desktop'.